src/Components/ActivationKey/ActivationKey.js

```diff
@@ -42,32 +36,27 @@ const ActivationKey = () => {
     error: keyError,
     data: activationKey,
   } = useActivationKey(id);
-  const { isLoading: areReleaseVersionsLoading, data: releaseVersions } =
-    useReleaseVersions();
-  const [isDeleteActivationKeyModalOpen, setIsDeleteActivationKeyModalOpen] =
-    useState(false);
+  const { data: releaseVersions } = useReleaseVersions();
+
   const [isEditActivationKeyModalOpen, setIsEditActivationKeyModalOpen] =
```

Comment:
Suggestion: Remove this state as the `EditActivationKeyModal` has been replaced by the wizard